post_processing/ci_plots.jl

```diff
@@ -0,0 +1,533 @@
+import CairoMakie
+import ClimaAnalysis
+import ClimaAnalysis: Visualize as viz
+import ClimaAnalysis: SimDir, slice_time, slice
+
+
+# From https://github.com/scheidan/PDFmerger.jl/blob/main/src/PDFmerger.jl
+# Licensed under MIT lincese
+import Base.Filesystem
+using Poppler_jll: pdfunite, pdfinfo, pdfseparate
+function merge_pdfs(
+    files::Vector{T},
+    destination::AbstractString = "merged.pdf";
```

Comment:
```suggestion
    destination::AbstractString = joinpath(common_dirname(files), "merged.pdf");
```
Could we do something like this, where `common_dirname` will grab the common directory among all files?

Reply:
Added